packages/react/src/components/timeInput/TimeInput.tsx

```diff
@@ -267,10 +267,12 @@ export const TimeInput = React.forwardRef<HTMLInputElement, TimeInputProps>(
 
     const hourInputId = `${id}-hours`;
     const minuteInputId = `${id}-minutes`;
+    const wrapperId = `${id}-hours`;
```

Comment:
Good catch! That would make sense in this case but I left it as it is because that id value (${id}-hours) is used for error, success, info and helper texts in InputWrapper component and I didn't dare to change the value because it could possibly break tests for users.

Reply:
The {${id}-hours} will be fixed in this [PR](https://github.com/City-of-Helsinki/helsinki-design-system/pull/841).
We need to sync these to PRs when is time for the release.

Reply:
Ah, yes.. It might get tricky to be sure that everything works as it should.

As for the `const wrapperId`, maybe I'll remove it and keep it as it was because it makes more sense to just pass the prop id as is to the InputWrapper? It's only used for the assistive text under the input.